srcpkgs/yank/template

```diff
@@ -1,17 +1,20 @@
 # Template file for 'yank'
 pkgname=yank
-version=1.2.0
+version=1.3.0
 revision=1
 build_style=gnu-makefile
 make_use_env=yes
-depends="xsel"
+make_build_args="YANKCMD=$(vopt_if wayland wl-copy xsel)"
+depends="$(vopt_if wayland wl-clipboard xsel)"
```

Comment:
hmm, this is unfortunate, isn't there a way to build with both xsel and wl-copy support?

Reply:
https://github.com/mptre/yank/blob/v1.3.0/yank.c#L470 hardcoded, gets resolved during compilation.